spec/features/show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe 'Homepage', type: :user do
+  fake_user = User.create(name: 'Kena', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  first_post = Post.create(author: fake_user, title: 'Hello', text: 'This is my first post')
+  second_post = Post.create(author: fake_user, title: 'Hello', text: 'This is my first post')
+  Post.create(author: fake_user, title: 'Hello', text: 'This is my first post')
+  Comment.create('text' => 'Bonjour les freres', 'author' => fake_user, 'post' => first_post)
+  Comment.create('text' => 'Bonjour les freres', 'author' => fake_user, 'post' => second_post)
+
+  it 'has a profile picture' do
+    visit "/users/#{fake_user.id}"
+
+    users = page.all('.profile-picture')
+    users.each do |profile_picture|
+      expect(profile_picture).to have_css('img')
+    end
+  end
+
```

Comment:
- [ ] @Team great work adding tests for the user show page however you forgot to add a test that asserts you can see the user's username.
- [ ] Kindly consider changing the file and description name for all integrated tests files, currently, it's difficult to know which file tests which page.